.github/workflows/reusable-macos.yml

```diff
@@ -69,4 +69,4 @@ jobs:
         --fail-on-improvement
         --path-prefix="./"
     - name: Tests
-      run: make test
+      run: make test TESTOPTS="-ugui"
```

Comment:
Rather than duplicating `make test TESTOPTS="-ugui"` in each file, would it make sense to add a `make citest` target in Makefile instead? I'm not sure about the name. Or maybe `make fastcitest`?

Reply:
What about `make ci`?

Reply:
`make ci` sounds good to me.